src/components/src/utils/AuthService.ts

```diff
@@ -156,11 +99,7 @@ export function logout() {
       window.dispatchEvent(new CustomEvent('authHeaderChanged', { detail: { authHeader: null, file: 'AuthService.3' } }));
       localStorage.removeItem(`${Config.cortexApi.scope}_oAuthUserName`);
       localStorage.removeItem(`${Config.cortexApi.scope}_b2bCart`);
-      localStorage.removeItem(`${Config.cortexApi.scope}_oAuthTokenAuthService`);
```

Comment:
I'm not seeing _oAuthTokenAuthService used anywhere. We should be able to remove it's usage